.env.example

```diff
@@ -1,4 +1,4 @@
-DEFI_ADAPTERS_LOG_LEVEL=debug
+DEFI_ADAPTERS_LOG_LEVEL=info
```

Comment:
This was interfering with the new-adapters cCLI command. It would dump debug lines after the first question is asked and before I had a chance to answer.